packages/main/src/SegmentedButtonItem.ts

```diff
@@ -164,6 +171,10 @@ class SegmentedButtonItem extends UI5Element implements IButton, ISegmentedButto
 		}
 	}
 
+	onAfterRendering(): void {
```

Comment:
can we move that on onBeforeRendering hook? otherwise it will trigger one more invalidation cycle.